core/src/main/scala/org/broadinstitute/dsde/rawls/bucketMigration/BucketMigrationServiceImpl.scala

```diff
@@ -170,7 +170,7 @@ class BucketMigrationServiceImpl(val dataSource: SlickDataSource, val samDAO: Sa
       // get migration progress if it exists
       workspacesWithProgressOpt <- workspaces.traverse { workspace =>
         dataSource
-          .inTransaction(getBucketMigrationProgress(workspace))
+          .inTransaction(dataAccess => getBucketMigrationProgress(workspace)(dataAccess))
```

Comment:
the previously-working function currying here needs to be explicit with the new `inTransaction` signatures.